src/database/migrations/20240411085039-add_auth_columns_to_users.js

```diff
@@ -16,11 +16,17 @@ module.exports = {
       allowNull: false,
       defaultValue: false,
     });
+    await queryInterface.addColumn('users', 'status', {
+      type: Sequelize.STRING,
+      allowNull: false,
+      defaultValue: 'Active',
```

Comment:
please if you have new task that require to add new column please create it's own migration